zerver/tests/test_subs.py

```diff
@@ -425,12 +424,12 @@ def delete_stream(self, stream, subscribed=True):
             self.assertEqual(deletion_events, [])
 
         with self.assertRaises(Stream.DoesNotExist):
-            Stream.objects.get(realm=get_realm("zulip"), name=active_name)
+            get_stream(stream_name=active_name, realm=get_realm("zulip"))
```

Comment:
I think this is a reason why Travis-CI check failing, `426` want this line raises `Stream.DoesNotExist` exception, but `get_stream()` is just returns `None` when there is no such stream. So the solution is revert this line back to the original form.